HC-937_Junior accounts (#62)
arduino/help-center-content: content/About Arduino/My Arduino Account/About-Arduino-accounts-for-minors.md

---
title: "About Arduino accounts for minors"
---

> Arduino offers two types of accounts for minors: <a href="#Junior-accounts">Junior accounts</a> and <a href="#Adult-accounts">Adult accounts for users between 14 and 16 years old</a>. They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.

Reviewer: ```suggestion
Arduino offers two types of accounts for minors: [Junior accounts](#Junior-accounts) and [Adult accounts for users between 14 and 16 years old](#Adult-accounts). They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
```
Markdown syntax works with anchor links.

Reviewer: * Markdown syntax works with anchor links.
* For me the first sentance was hard to follow, I would suggest adding some structure:

```suggestion
Arduino offers two types of accounts for minors:

* [Junior accounts](#Junior-accounts). For users below 14 years old.
* [Adult accounts with restrictions](#Adult-accounts). For users between 14 and 16 years old.

They differ in the features that users can access and the way that Arduino handles [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy). Both account types require the minor's account to be linked to a person exercising parental authority. Learn more about Arduino accounts for minors below.
```

> <h2 id="Junior-accounts"></h2>

Reviewer: ```suggestion
<a id="Junior-accounts"></a>
```
I'm not sure if either is considered good practise but this is what I've been using to add "anchor targets".

To complete the registration of a Junior Account, it is necessary to provide a parent's email. Arduino will process the [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy) of the person exercising parental authority.

> If the parent's email is not provided the Junior Account will not be created and all data provided will be deleted within 30 days.

**With a Junior Account, users are able to:**

* Access Educational products and services intended for middle school students (view the full list on [Arduino Education](https://www.arduino.cc/education)).
* Download and use Arduino IDE.
* Download and use the Arduino Science Journal App. Use the Science Journal Apps only for creating experiments saved locally in the App. Sharing functionality with Google Drive is not made available to Junior Accounts.
* Submit Assignment from the Science Journal App to Google Classroom, subject to an existing plan held by your teacher and approval of the latter.

### Upgrading to an Adult account

After turning 14, users of Junior accounts will be prompt to upgrade to an <a href="#Adult-accounts">Adult account for users between 14 and 16 years old</a>.

> ![Upgrading to regular account pop-up](img/Upgrade_to_regular_account.png)

Reviewer: The image is very large, I would scale it down to 600 px width.

> This process will move all their files, projects & data to the upgraded account, but the account will still require a person exercising parental authority.

Reviewer: ```suggestion
This will transfer all their files, projects and data to the upgraded account, but the account will still require a person exercising parental authority.
```

> <h2 id="Adult-accounts"></h2>

Reviewer: ```suggestion
<a id="Adult-accounts"></a>
```

## Adult accounts for users between 14 and 16 years old

As for the Junior Accounts, users between 14 and 16 years old will be asked to provide the email of the person exercising parental authority. Arduino will process the [personal data](https://www.arduino.cc/en/privacy-policy/children-privacy-policy) of the latter.

> Through the Adult Account for users between 14 and 16 years old, the user has access to all the services available on the Arduino website, except for those services/features expressly reserved for the person exercising parental authority. Only the person exercising parental authority may choose which services would be activated and which services the user between 14 and 16 years old can access.

Reviewer: ```suggestion
With an Adult Account for users between 14 and 16 years old, the user has access to all the services available on the Arduino website, except for those services and features expressly reserved for the person exercising parental authority. Only the person exercising parental authority may choose which services would be activated and which services the user between 14 and 16 years old can access.
```

> ## Useful information

Reviewer: ```suggestion
## Further reading
```

---
title: "About Arduino accounts for minors"
---

Arduino offers two types of accounts for minors:

* [Junior accounts](#Junior-accounts). For users below 14 years old.

> * [Adult accounts with restrictions](#Adult-accounts). For users between 14 and 16 years old.

Reviewer: Perhaps we should call them "regular" accounts instead of "adult".

This is how they're referred to in the upgrade prompt, and it makes sense since 14 year olds are not adults.

![image](https://user-images.githubusercontent.com/71350948/166651642-516f7871-2f62-435f-bcde-c22eca5b2d72.png)

Author: Yes, I agree, it is less confusing that way since the image calls it a "regular account". My only concern is that in Arduino's privacy policy, these accounts are referred to as "Adult account (for users between 14 and 16 years old" and this can create confusion.

Author: Updated [b55cbb9](https://github.com/arduino/help-center-content/pull/62/commits/b55cbb91f85b8f5c489bf267b539df5efaf7527a)

Reviewer: @Renat0Ribeir0 Ah, then I see where you were coming from. But yeah on the whole I think it's a change for the better.
